schemas/test/en/test_numbers.json

```diff
@@ -187,11 +187,11 @@
                                     },
                                     {
                                         "id": "test-min",
-                                        "label": "Min Test (123 to 9,999,999,999)",
+                                        "label": "Min Test (-123 to 9,999,999,999,999)",
```

Comment:
the enforced limit in this test is actually -123 to 999,999,999,999,999, not -123 to 9,999,999,999,999

Reply:
Updated 👍